src/main/java/by/intexsoft/ignatieva/Printer.java

```diff
@@ -0,0 +1,11 @@
+package by.intexsoft.ignatieva;
+
+import java.util.Arrays;
+
+public class Printer {
+    public static void print(int[][] matrix) {
+        for (int i = 0; i < matrix.length; i++) {
```

Comment:
when you don't need index of current element - it's beter to use foreach loop like:
```
for (String item : someList) {
    System.out.println(item);
}
```